packages/elements/src/slider/elements/slider-marker.ts

```diff
@@ -0,0 +1,71 @@
+import { BasicElement, CSSResultGroup, TemplateResult, css, html } from '@refinitiv-ui/core';
+import { customElement } from '@refinitiv-ui/core/decorators/custom-element.js';
+import { property } from '@refinitiv-ui/core/decorators/property.js';
+
+import { VERSION } from '../../version.js';
+
+@customElement('ef-slider-marker')
+export class SliderMarker extends BasicElement {
+  /**
+   * Element version number
+   * @returns version number
+   */
+  static override get version(): string {
+    return VERSION;
+  }
+
+  static override get styles(): CSSResultGroup {
+    return css`
+      :host {
+        position: absolute;
+      }
+      [part='marker'] {
+        width: 100%;
+        height: 100%;
+      }
+      [part='label'] {
+        white-space: nowrap;
+        position: absolute;
+      }
+    `;
+  }
+
+  /**
+   * Hide the element from accessability API,
+   * as it could be detrimental to assistive technology
+   */
+  @property({ type: String, reflect: true, attribute: 'aria-hidden' })
+  public override ariaHidden = 'true';
```

Comment:
We never rely on using AriaMixins from typescript that derived from AOM, https://github.com/WICG/aom. This is because of highly unstable specification and subject to change. We usually simply use `setAttribute` in `firstUpdated` to achieve this.

Reply:
Firefox has implemented the feature recently. It's supported on all major browsers now. 
https://developer.mozilla.org/en-US/docs/Web/API/Element/ariaHidden
https://caniuse.com/?search=ariahidden

Reply:
Per our discussion, let's hard code `aria-hidden` value as `true` in `firstUpdated()` life cycle callback instead.